Rakefile

```
@@ -155,3 +155,26 @@ namespace :release do
     end
   end
 end
+
+namespace :pkg do
+  desc 'Sync the root rust-toolchain version to all crates'
+  task :'rust_version:sync' do
+    rust_version = File.open('rust-toolchain').read.chomp
```

We can avoid possibly leaking a file handle by doing this:

```suggestion
    rust_version = File.read('rust-toolchain').chomp
```